anyway/llm.py

```diff
@@ -0,0 +1,83 @@
+from openai import OpenAI
+import json
+import tiktoken
+
+from langchain.output_parsers import EnumOutputParser
+from langchain_core.prompts import PromptTemplate
+from langchain_openai import ChatOpenAI
+import langchain
+from enum import Enum
+from anyway import secrets
+
+api_key = secrets.get("OPENAI_API_KEY")
```

Comment:
I suggest inserting the secret into a function, to avoid tests failures in tests from forks